com.github.debauchee.barrier.json

```diff
@@ -77,6 +77,9 @@
         {
             "name" : "barrier",
             "buildsystem" : "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
```

Comment:
The environment already sets CFLAGS for optimization so this might not be necessary. If so (because the build system does something different when set) `RelWithDebInfo` is preferred.

Reply:
I changed this mostly to conform with our other builds (macOS, Windows). If it's not needed, I can revert it.

Reply:
reverted